src/Controller/FrontendController.php

```diff
@@ -39,6 +39,29 @@ public function __construct(BoxConfigurationRepository $boxConfigurationReposito
         $this->tokenService = $tokenService;
     }
 
+    /**
+     * @Route("/fake-online", name="fake_online", condition="env('string:APP_ENV') === 'dev'"))
+     *
+     * Simulates an unstable FBS endpoint for 99 requests.
+     *
+     * @return Response
+     *   Http response
+     *
+     * @throws \Exception
+     */
+    public function fakeOnline()
```

Comment:
????? Should this be in tests or something?

Reply:
I'll move it.